src/rpc/rawtransaction.cpp

```diff
@@ -218,6 +218,252 @@ UniValue getrawtransaction(const UniValue &params, bool fHelp)
     return result;
 }
 
+UniValue getrawblocktransactions(const UniValue &params, bool fHelp)
+{
+    if (fHelp || params.size() < 1 || params.size() > 2)
+        throw runtime_error(
+            "getrawtransactions \"blockhash\" ( verbose )\n"
```

Comment:
help needs to be "getrawblocktransactions".

I would like to move away from the boolean flag system that some bitcoin RPCs currently use because it is not user friendly.  Although it may be a little weird when writing code, I propose we use the command line convention of "-flag" or "--flag=value" (all the dash options MUST be optional), placed before the other args.  Another possibility is to use "flag=value" which is kind of a compromise between the programmatic way of specifying keyword arguments and the command line.

Reply:
will fix that error in help.